pkg/controller/utils/constants.go

```diff
@@ -3,4 +3,7 @@ package utils
 const (
 	// LastBackupAnnotation is the Kubernetes Annotation key that holds the object checksum from the last backup.
 	LastBackupAnnotation = "hive.openshift.io/last-backup-checksum"
+
+	// VeleroBackupEnvVar is the name of the environment variable used to tell the controller manager to enable velero backup integration.
+	VeleroBackupEnvVar = "HIVE_VELERO_BACKUP"
```

Comment:
It is used outside of controllers i.e. in  pkg/operator/hive/hive.go . IMO it should go to https://github.com/openshift/hive/blob/master/pkg/constants/constants.go

Reply:
You're right, thanks for the comment. Will move.